src/main/java/repository/DAOUser.java

```diff
@@ -0,0 +1,71 @@
+package repository;
+
+import models.User;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DAOUser {
+
+    private static final String SQL_COMMAND_ADD = "INSERT INTO _user (id, name, creation_date) VALUE (?, ?, ?)";
+    private static final String SQL_COMMAND_GET_BY_ID = "SELECT * FROM _user WHERE id = ?;";
+    private static final String SQL_COMMAND_UPDATE = "UPDATE _user SET id = ?, name = ?, creation_date = ? WHERE id = ?";
+    private static final String SQL_COMMAND_DELETE = "DELETE FROM _user WHERE id = ?";
+
+
+    public void addUser(User user) {
+        try (Connection connection = ConnectionConfig.getConnection();
+             PreparedStatement statement = connection.prepareStatement(SQL_COMMAND_ADD)) {
+            statement.setString(1, String.valueOf(user.getId()));
+            statement.setString(2, user.getName());
+            statement.setString(3, String.valueOf(user.getTimeCreationUser()));
+        } catch (SQLException | IOException e) {
```

Comment:
db here will not be updated. you need explicitly call method `statement.executeUpdate();` after setting all value to query